config/clientlb.ini

```diff
@@ -0,0 +1,36 @@
+[DEFAULT]
+create_request_rate=100
+max_requests=inf
+sleep_time=0.00001
```

Comment:
Be careful with small values for **_sleep_time_** this can cause slowness in the simulation.

Reply:
Well remembered.